rest-service/manager_rest/resource_manager.py

```diff
@@ -1398,6 +1398,14 @@ def get_deployment_parents_from_labels(labels):
                 parents.append(value)
         return parents
 
+    @staticmethod
+    def get_deployment_object_type_from_labels(labels):
+        types = set()
+        for label, value in labels:
```

Comment:
Seme comment as in last PR, it's better to call it `key` instead of `label`, as ` `label` defines the whole object :)

Reply:
yeah, my bad will update that